pom.xml

```diff
@@ -202,6 +202,7 @@
         <clj-time.version>0.8.0</clj-time.version>
         <curator.version>2.9.0</curator.version>
         <json-simple.version>1.1</json-simple.version>
+        <msgpack.version>0.6.12</msgpack.version>
```

Comment:
This would be *much* more useful if it were implemented using a newer version of the `msgpack-core` library—they changed the name after 0.6.12—because 0.7 and above supports the `BINARY` format, which lets you send arbitrary bytes.  Without that, you won't be able to send tuples containing arbitrary bytes with this serializer.  This is also a problem with the JSON serializer (because JSON strings can't contain non-Unicode characters), but it would be great if we didn't have the problem here.